app/utils/trackableEvents.js

```diff
@@ -0,0 +1,18 @@
+export const JOIN_A_WORKSPACE = 'join_a_workspace';
```

Comment:
Create a file `app/utils/log/index.js` with all exports.
This way you can import `logEvent` and `events` from the same source.

Reply:
I've placed all the `log.js` code inside the `log/index.js` together with the export for the events.